test/run_migration_test

```diff
@@ -53,4 +53,17 @@ assert_migration_succeeds ()
     container_from= container_to=
 }
 
+# test case suggested in https://github.com/sclorg/postgresql-container/pull/521
+assert_migration_fails()
+{
+    docker run --rm -e POSTGRESQL_MIGRATION_REMOTE_HOST=invalid-host-name -e POSTGRESQL_MIGRATION_ADMIN_PASSWORD=password "$image_to" run-postgresql --version
```

Comment:
Did not notice we still have `set -e` in here. This will need to be moved directly into the if statement below to not trigger early exit.

Reply:
Good point, thanks for it.